### ISSUE TITLE
Logo image does not have alt text

### ISSUE THREAD

**Issue by dengeist** [jhwheeler/simple-spirit#2]
Alternate text is required by [the HTML5 `<img>` tag spec](https://dev.w3.org/html5/spec-preview/the-img-element.html#general-guidelines), so any images without will not pass validation. Additionally, alternate text for image contents is [a WCAG 2.0 Level A success criterion](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#text-equiv-all):

> **1.1.1 Non-text Content:** All non-text content that is presented to the user has a text alternative that serves the equivalent purpose [...]

In pursuit of this, make sure this image and all your images have appropriate alternate text content if they are important to the meaning of your application. Only when your image is NOT important--it is purely decorative and would hold no specific meaning to sighted or unsighted users--would you be able to skip the `alt`. You can see the above links for discussions of good and bad a11y practices.

Particularly relevant here: 

> Many pages include logos, insignia, flags, or emblems, which stand for a particular entity such as a company, organization, project, band, software package, country, or some such.
> 
> If the logo is being used to represent the entity, e.g. as a page heading, the alt attribute must contain the name of the entity being represented by the logo. The `alt` attribute *must not* contain text like the word "logo", as it is not the fact that it is a logo that is being conveyed, it's the entity itself.
>
> If the logo is being used next to the name of the entity that it represents, then the logo is supplemental, and its `alt` attribute must instead be empty.

As well, from [the H30 technique](https://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/H30.html) of the WCAG,

> When an image is the only content of a link, the text alternative for the image describes the unique function of the link.

That means that sometimes the `alt` attribute may not be sufficient, but you can enhance it for screen readers with [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute). As with many usability concerns, accessibly delivering content is not always a cut-and-dry issue, but here's what your logo might look like with all the above in mind:

```html
 <a href="/" aria-label="Simple Spirit homepage">
       <img src="simple-spirit-logo.png" alt="Simple Spirit">
</a>
```

**Comment by jhwheeler** [comment]
Noted, will do. Thank you for the resources and references!

**Comment by jhwheeler** [comment]
Fixed! Thanks :+1:
